From 66afe4f18c6184c5f786498532d8d996d8f8fc5f Mon Sep 17 00:00:00 2001
From: Olivier Fourdan <olivier.fourdan@intel.com>
Date: Fri, 19 Apr 2013 10:02:45 +0200
Subject: build: Add path to NDK 4.7 toolchain

BZ: 103828

Add the path to NDK 4.7 toolchain so we can use it
once it's made available.

Change-Id: I8cf8bbd4ac2ff13db361d0bd9fcf34c786d5b433
Orig-Change-Id: Id139d3264fed7bfacf32ba9cc28e1fff75e166e1
Category: device enablement
Domain: AOSP
Origin: internal
Upstream-Candidate: no
Signed-off-by: Olivier Fourdan <olivier.fourdan@intel.com>
---
 core/dumpvar.mk |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/core/dumpvar.mk b/core/dumpvar.mk
index 92bf6af..d1567b8 100644
--- a/core/dumpvar.mk
+++ b/core/dumpvar.mk
@@ -26,6 +26,12 @@ else ifeq ($(TARGET_ARCH),x86)
         # this should be copied to HOST_OUT_EXECUTABLES instead
         ABP:=$(ABP):$(PWD)/prebuilts/gcc/$(HOST_PREBUILT_TAG)/x86/i686-linux-android-$(TARGET_GCC_VERSION)/bin
     endif
+
+# Add the x86_64 toolchain bin dir if it actually exists
+    ifneq ($(wildcard $(PWD)/prebuilts/gcc/linux-x86/x86/x86_64-linux-android-4.7/bin),)
+        # this should be copied to HOST_OUT_EXECUTABLES instead
+        ABP:=$(ABP):$(PWD)/prebuilts/gcc/linux-x86/x86/x86_64-linux-android-4.7/bin
+    endif
 endif
 
 # Add the mips toolchain bin dir if it actually exists
-- 
1.7.4.1

